art/attacks/inference/attribute_inference/baseline.py

```diff
@@ -111,6 +111,10 @@ def __init__(
         self._check_params()
         self.attack_feature = get_feature_index(self.attack_feature)
 
+        self._nb_classes: Optional[int] = None
+        if self._values is not None:
+            self._nb_classes = len(self._values)
```

Comment:
Do we need these two lines? Or can we document the definition with `len(self._values)` in a comment in line 114?